### ISSUE TITLE
lua 5.3 support?

### ISSUE THREAD

**Issue by Stretto** [JakobOvrum/LuaD#106]
It's been a while... or are there any other maintained lua bindings for D that are up to date?

**Comment by Soulsbane** [comment]
Yeah I wish we could at least get a release with semver so dub will stop complaining on every build:

WARNING: A deprecated branch based version specification is used for the dependency luad. Please use numbered versions instead. Also note that you can still use the dub.selections.json file to override a certain dependency to use a branch instead.
